web-client/src/views/FileDocument/StateDrivenFileInput.tsx

```diff
@@ -1,22 +1,21 @@
-import { Button } from '../../ustc-ui/Button/Button';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { FileInput } from './FileInput';
 import { cloneFile } from '../FileHandlingHelpers/cloneFile';
-import { connect } from '@web-client/presenter/shared.cerebral';
```

Comment:
Our tsconfig is set up to allow to prefer the `@web-client` etc aliasing. You can have vscode use this for auto-imports by adding ` "typescript.preferences.importModuleSpecifier": "non-relative"` to your vsCode settings

Reply:
Homeboy uses IntelliJ. That setting looks like this for us:
<img width="1485" height="995" alt="Screenshot 2025-12-19 at 11 36 01 AM" src="https://github.com/user-attachments/assets/a5348688-8550-47b4-bb59-97672b9b2937" />
(Uncheck the "Use paths relative to tsconfig.json" option)